src/sentry/api/helpers/group_index/delete.py

```diff
@@ -116,7 +135,8 @@ def create_audit_entries(
             "object.delete.queued",
             extra={
                 "object_id": group.id,
-                "organization_id": project.organization_id,
+                "project_id": group.project_id,
+                "organization_slug": group.project.organization.slug,
```

Comment:
The org slug is more readily accessible to us since it shows in the URL of a customer's account.

Reply:
woiuldn't this make another sql query to fetch this relationship

Reply:
@scttcper I have addressed your feedback by removing the extra call. I should have enough with the project ID.